app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17, \
+        "sum of returned list should be equal value"
```

Comment:
You are creating an unnecessary variable in your assert statement. You should directly compare the result of the function call to the expected result. Refer to the Code Style section of the checklist.